.test-infra/jenkins/JavaTestProperties.groovy

```diff
@@ -17,10 +17,5 @@
  */
 
 class JavaTestProperties {
-  final static List<String> SUPPORTED_CONTAINER_TASKS = [
-    'java8',
-    'java11',
-    'java17',
-    'java21'
```

Comment:
this line causes Jenkins trying to build Java21 container as part of publish SDK snapshot job